### DISCUSSION TITLE
IP Pools, what's the RDNS AND DNS SETUP

### DISCUSSION THREAD

**Discussion by traderpedroso** [postalserver/postal#2261] [Help with using Postal] [answered]
I have 2 IP on the IP pool, every IP needs to set i hostname

Example on postal webgui for IP POOLS

1.1.1.1   ip1.example.com
2.2.2.2 ip2.example.com

my PTR RDNS is postal.example.com

My DNS pointing on Cloudflare  

postal.example.com to 1.1.1.1
ip1.example.com 1.1.1.1
 ip2.example.com 2.2.2.2

Your IP address 1.1.1.1 is associated with the postal.example.com
Nevertheless, your message appears to be sent from ip1.example.com.

You may want to change your pointer (PTR type) DNS record and the host name of your server to the same value.

```

Here are the tested values for this check:
IP: 1.1.1.1
HELO: ip1.example.com
rDNS: postal.example.com
```

So on single IP using postal.example.com it's working perfect but when try to use 2 IP  Your reverse DNS does not match with your sending domain.

**Comment by willpower232** [comment, marked as answer]
We're not using IP Pools but our setup is similar to yours.

postal.example.com is 1.1.1.1
first.postal.example.com is 1.1.1.1
the reverse dns for 1.1.1.1 is first.postal.example.com

maybe using a double subdomain is helping out in our case?

**Reply by traderpedroso** [reply]
> We're not using IP Pools but our setup is similar to yours.
> 
> postal.example.com is 1.1.1.1 first.postal.example.com is 1.1.1.1 the reverse dns for 1.1.1.1 is first.postal.example.com
> 
> maybe using a double subdomain is helping out in our case?

Found the solution for that the problem was the reverse DNS on host  I set main IP as mail. And the second as email. Config the SPF for was every thing working but not full authenticate because was missing setup reverse DNS for every IP on the host different one from other
